### PR TITLE
📝 Document new OAuth scopes

### PR DESCRIPTION
**Pull request by coyotte508**
cc @Wauplin @Pierrci @julien-c 

Those will be available at next HF deploy

### CODE REVIEW

File: docs/hub/spaces-config-reference.md [hunk 1]
```diff
@@ -74,6 +74,9 @@ Whether a connected OAuth app is associated to this Space. See [Adding a Sign-In
 **`hf_oauth_redirect_path`** : _string_  
 Authorized relative redirect path of the connected OAuth app. `/login/callback` and `/auth/callback` are authorized by default and do not need this parameter. See [Adding a Sign-In with HF button to your space](https://huggingface.co/docs/hub/spaces-oauth) for more details.
 
+**`hf_oauth_scopes`** : _List[_string_]
```
Comment (julien-c) [inline docs/hub/spaces-config-reference.md:77]: is it a list of string, or a space-separated string?

File: docs/hub/spaces-oauth.md [hunk 2]
```diff
@@ -60,12 +61,18 @@ You can add a custom relative redirect path by setting `hf_oauth_redirect_path`
...
 ## Scopes
 
-The following scopes are available:
+The following scopes are always included for Spaces:
 
 - `openid`: Get the ID token in addition to the access token.
 - `profile`: Get the user's profile information (username, avatar, etc.)
 
-You should use `"openid profile"` as the scope for your OAuth app.
+Those scopes are optional and can be added by setting `hf_oauth_scopes` in your Space's metadata:
```
Comment (julien-c) [inline docs/hub/spaces-oauth.md:69]:
```suggestion
Those scopes are optional and can be added by setting `hf_oauth_scopes` in your Space's metadata (as a space-separated string):
```

**Review by Wauplin — APPROVED** [review]
So cool!

File: docs/hub/spaces-config-reference.md (hunk 1 above)
Comment (pcuenca) [inline docs/hub/spaces-config-reference.md:77]:
```suggestion
**`hf_oauth_scopes`** : _List[string]_
```

File: docs/hub/oauth.md [hunk 3]
```diff
@@ -31,12 +31,16 @@ The currently supported scopes are:
 - `openid`: Get the ID token in addition to the access token.
 - `profile`: Get the user's profile information (username, avatar, etc.)
 - `email`: Get the user's email address.
+- `read-repos`: Get read access to a user's personal repos.
```
Comment (pcuenca) [inline docs/hub/oauth.md:34]:
```suggestion
- `read-repos`: Get read access to the user's personal repos.
```

File: docs/hub/oauth.md [hunk 4]
```diff
@@ -31,12 +31,16 @@ The currently supported scopes are:
...
+- `write-repos`: Get write access to a user's personal repos. Does not grant read access on its own, you need to include `read-repos` as well.
```
Comment (pcuenca) [inline docs/hub/oauth.md:35]:
```suggestion
- `write-repos`: Get write access to the user's personal repos. Does not grant read access on its own, you need to include `read-repos` as well.
```

File: docs/hub/spaces-oauth.md [hunk 5]
```diff
@@ -60,12 +61,18 @@ You can add a custom relative redirect path by setting `hf_oauth_redirect_path`
...
+
+- `email`: Get the user's email address.
+- `read-repos`: Get read access to a user's personal repos.
```
Comment (pcuenca) [inline docs/hub/spaces-oauth.md:72]:
```suggestion
- `read-repos`: Get read access to the user's personal repos.
```

File: docs/hub/spaces-oauth.md [hunk 6]
```diff
@@ -60,12 +61,18 @@ You can add a custom relative redirect path by setting `hf_oauth_redirect_path`
...
+- `write-repos`: Get write access to a user's personal repos. Does not grant read access on its own, you need to include `read-repos` as well.
```
Comment (pcuenca) [inline docs/hub/spaces-oauth.md:73]:
```suggestion
- `write-repos`: Get write access to the user's personal repos. Does not grant read access on its own, you need to include `read-repos` as well.
```

**Review by pcuenca — COMMENTED** [review]
Looks good!

File: docs/hub/spaces-oauth.md (hunk 2 above)
Comment (Pierrci) [reply docs/hub/spaces-oauth.md:69]: then the example at https://github.com/huggingface/hub-docs/pull/1134/files#diff-8c77839fc5d40b69830303ec9567fbc0a4fbbefabf3309abef7de83f758f2652R37 is wrong, no?

**Pull request merged**